lib/ansible/modules/windows/win_domain_computer.ps1

```diff
@@ -0,0 +1,218 @@
+#!powershell
+#
+# Copyright 2017, AMTEGA - Xunta de Galicia
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Ansible "directives"
+# WANT_JSON
```

Comment:
We no longer do it like this.
Please use:
```yaml
#Requires -Module Ansible.ModuleUtils.Legacy
```

Reply:
changed